pyproject.toml

```diff
@@ -90,11 +90,11 @@ dependencies = [
   "colorama<1",
   "colorlog",
   "crash",
-  "crate[sqlalchemy]>=0.34",
+  "crate @ git+https://github.com/crate/crate-python.git@no-sqlalchemy",
```

Comment:
The `crate` package will probably need to be published before being able to conclude the migration. For example, let's use/ focus on `crate[sqlalchemy]`.

Reply:
Published crate-python@no-sqlalchemy [^1] as crate-1.0.0.dev0 [^2], in order to satisfy downsteam dependencies, without tripping any existing users of `crate`.

[^1]: https://github.com/crate/crate-python/tree/no-sqlalchemy
[^2]: https://pypi.org/project/crate/1.0.0.dev0/